bin/jaxb2ruby

```diff
@@ -81,5 +81,5 @@ begin
     File.open(File.join(outdir, klass.path), "w") { |io| io.puts template.build(klass) }
   end
 rescue => e
-  abort "class generation failed: #{e}"
+  abort "class generation failed: #{e} \n#{'*' * 100}\nbacktrace: \n#{ e.backtrace.join("\n") }"
```

Comment:
Not sure how I feel about this. I don't think command line programs should spew stacktraces to stderr. Really the only exceptions that should be caught here are `JAXB2Ruby::Error`s. If it's anything else then it's a bug (hence this pull request). 

Maybe a debug that printed what the program was doing would be better?